.storybook/components/AuthenticatedPageTemplate.tsx

```diff
@@ -0,0 +1,81 @@
+import { PropsWithChildren, useMemo } from 'react';
+import { Logo } from '@ag.ds-next/react/ag-branding';
+import { tokens } from '@ag.ds-next/react/core';
+import { LinkList } from '@ag.ds-next/react/link-list';
+import { Text } from '@ag.ds-next/react/text';
+import {
+	AppLayout,
+	AppLayoutSidebar,
+	AppLayoutHeader,
+	AppLayoutHeaderTitles,
+	AppLayoutHeaderUserLink,
+	AppLayoutFooter,
+	AppLayoutFooterDivider,
+	AppLayoutContent,
+} from '@ag.ds-next/react/app-layout';
+
+type PageTemplateProps = PropsWithChildren<{
+	background?: 'body' | 'bodyAlt';
+}>;
+
+export function AuthenticatedPageTemplate({
```

Comment:
This is a component that we can use to test the kitchen sink and other full page stories with the app layout. Note: We have another component in this same folder already called `.storybook/components/PageTemplate.tsx`